server/api/users.js

```diff
@@ -1,13 +1,24 @@
 const express = require('express')
 const router = express.Router()
-const { User, Session } = require('../db')
+const { User, Session, Cart, CartItem } = require('../db')
 const bcrypt = require('bcrypt')
 
 const A_WEEK_IN_SECONDS = 60 * 60 * 24 * 7;
```

Comment:
Add to constants file?

Reply:
We could. This is the only place it's used, though.